### ISSUE TITLE
Add flexibility in the paths output to combined coverage reports

### ISSUE THREAD

**Issue by charlesluzar** [verasana/combine_coverage#2]
> This is a good PR if it makes the coverage extension. It is even better if the coverage report is relative to the root rather than the absolute file path.  The current output is not portable across file system locations.

_Originally posted by @freemansoft in https://github.com/verasana/combine_coverage/issues/1#issuecomment-1919932646_
            
Making output paths relative to the repo root makes a lot of sense... so much sense that I feel there was a reason I didn't take that route initially. I want to think about the ramifications and either always use paths relative to the root or make it easy to switch between the two via a flag. Open to input here as well.

**Comment by freemansoft** [comment]
Maybe some of the HTML or other tools need an absolute path to the source files for navigation?

**Comment by freemansoft** [comment]
Just checked out 1.0.1 on my local machine.  It changed the output from full absolute path to the path relative to the top, where it was run from.

```
--repo-path=.
```
and 
```
--repo-path=$PWD
```
both root the entries coverage file as `/` instead of the full absolute path.

This works for me in Azure DevOps with their PublishCodeCoverageResult@1 after converting the lcov.info to cobertura.

**Comment by charlesluzar** [comment]
> Maybe some of the HTML or other tools need an absolute path to the source files for navigation?

That's it! Thanks for the memory jog. I was running genhtml locally and genhtml did not play nicely with relative paths, you had to have absolute paths. Now, we ended up just kicking everything to Coveralls anyway so it ended up being somewhat moot.

Having said that, I don't know we need to offer easy absolute paths until someone actually asks for it. Among the 3 people paying attention to this package, all of us are using relative paths. So, there it will stay. 😆 

Closing this as unplanned.
